source/deployment/sso-saml-okta.rst

```diff
@@ -44,30 +42,38 @@ If you’d like to use a Mattermost logo for the application, you can download o
 
 	.. image:: ../../source/images/okta_5_advanced_saml_settings.PNG
 
-9. Enter attribute statements, which will be used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`. Email and username attributes are required. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
+9. Enter attribute statements used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our `documentation on SAML configuration settings <https://docs.mattermost.com/administration/config-settings.html#saml>`__. Email and username attributes are required. For SAML with Okta, an `ID attribute <https://docs.mattermost.com/administration/config-settings.html#id17>`__ is also required, and that ID must be mapped to ``user.id``. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
 
-	.. image:: ../../source/images/okta_6_attribute_statements.PNG
+	.. image:: ../../source/images/okta_6_attribute_statements.png
```

Comment:
Need to confirm that the extension is lowercase as it doesn't look like it's showing in preview now.

Reply:
There were 2 files with identical file names (except that one extension was PNG and the other was png). I've removed the older file as part of this PR.

Reply:
Confirmed that the correct screenshot is displaying now.